From 11115e08eae323df49aedd511d19627d1617a777 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 15 Sep 2014 10:52:32 +0200
Subject: [PATCH 072/192] net/cert: increase default key length

Should we add something for chrome://settings, or just leave
it as hidden as it was before?

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 net/cert/x509_util.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/cert/x509_util.cc b/net/cert/x509_util.cc
index c3f26c8c9343d..a1a07d4bf85d2 100644
--- a/net/cert/x509_util.cc
+++ b/net/cert/x509_util.cc
@@ -254,7 +254,7 @@ bool GetTLSServerEndPointChannelBinding(const X509Certificate& certificate,
 }
 
 // RSA keys created by CreateKeyAndSelfSignedCert will be of this length.
-static const uint16_t kRSAKeyLength = 1024;
+static const uint16_t kRSAKeyLength = 2048;
 
 // Certificates made by CreateKeyAndSelfSignedCert will be signed using this
 // digest algorithm.
-- 
2.34.1

